### PR TITLE
Hebrew transaltion -  2.3.1.0 Support

### PR DESCRIPTION
**Pull request by HypeGameMode**
Added Hebrew translation for the newer update 2.3.1.0

### CODE REVIEW

**Comment by micwoj92** [comment]
@dinuto Do you agree with these translations?

**Review by dinuto — CHANGES_REQUESTED** [review]

**Comment by HypeGameMode** [comment]
That's good for me.
You can merge it

**Comment by micwoj92** [comment]
> That's good for me.
> You can merge it

Merge now? There is mistake, for example in lines 69-70.

**Comment by dinuto** [comment]
I think you have to manually add the changes I suggested (if you agree to them, of course).
<div dir="rtl">סורי על הסיבוך, גם אני חשבתי שאפשר פשוט לאשר את ההצעות והן יתווספו אוטומטית...</div>

**Comment by HypeGameMode** [comment]
> 
> 
> > That's good for me.
> > You can merge it
> 
> Merge now? There is mistake, for example in lines 69-70.

Take all @dinuto requests and then merge it.
He fixed some mistakes I did in the translation, so it's good enough what he did

**Comment by micwoj92** [comment]
Ok, I will do that now. You marked these as resolved so I didn't exactly know what you meant :/

File: Langs/040D.lang [hunk 1]
```diff
@@ -57,3 +58,19 @@ Invalid Release Tags Received!=קיימת שגיאה במציאת טאג עם ג
...
+Done=בוצע
+Loading WMIC=WMIC טוען 
+Warning=אזהרה
+[Descriptions]
+Architecture=.מספר הנתונים שהמעבד ומערכת ההפעלה שלך יכולה לעבד באותו הזמן. מערכת הפעלה עם 32 סיביות דורשת פרמוט והתקנת ווינדוס חדש. מעבד 32 סיביות דורש החלפת מעבד
+Boot=.ניתנת לשינוי בלוחות אם חדשים דרך הגדרות הביוס. לפי הוראות היצרן Legacy שיטה בה משתמש הלוח אם כדי להעלות את הווינדוס. שיטת 
+CPU Name=.המעבד שמותקן במחשב. תאימות המעבד עלולה להשתנות. ניתנת להחלפה במחשבים נייחים; לא ניתנת להחלפה במחשבים ניידים
+CPU Cores=.מספר המשימות שהמעבד יכול לבצע בו זמנית. ניתן להחלפה במחשבים נייחים; לא ניתן להחלפה במחשבים ניידים
+CPU Speed=.הקצב שהמעבד מבצע משימות. ניתן להחלפה במחשבים נייחים; לא ניתן להחלפה במחשבים ניידים
+רמת הפיטצ'רים שהכרטיס מסך תומך/DirectX-רמת ה
```
[the remark this answers is not in the document]
Comment (micwoj92) [reply Langs/040D.lang:70]: So this line should be deleted? That's last thing to check before merging.

[the remark this answers is not in the document]
Comment (dinuto) [reply Langs/040D.lang:70]: Yes, thank you!

**Pull request merged**